modules/ROOT/pages/whatsnew.adoc

```diff
@@ -20,57 +20,29 @@ CAUTION: {sgw-t} 3.0.x introduces some breaking changes.
 If you are upgrading from 2.x, please refer to the {upgrading--xref} page.
 Users should be able to upgrade to 3.1.x from 3.0.x without manual intervention.
 
+* *New support for Scopes and Collections* +
+Couchbase has introduced support for Scopes and Collections for self-managed cloud-to-edge deployments only in xref:couchbase-lite:ROOT:index.adoc[Couchbase Lite 3.1.0] and xref:introduction.adoc[Sync Gateway 3.1.0]. 
+This release won't cause any issues with existing apps, as it's compatible with older versions. 
+If you have an app that uses bucket-based APIs, you can still upgrade to 3.1, but please note that this API is now deprecated.
 
-== Overview of {sgw-t} 3.1.0
+* *Improved Data Organization and Access Control for Scopes and Collections* +
+Couchbase Mobile now offers Scopes and Collections, allowing more efficient and scalable data organisation within a bucket. 
+This also introduces an improved method of defining and enforcing data xref:access-control-concepts.adoc[Access Control] more granularly. 
+Multi-tenant apps will also experience better scalability and independent data lifecycle management.
 
-// tag::scopes-and-collections[]
```

Comment:
is this tag included from anywhere else?

Reply:
`modules/ROOT/pages/_partials/release_notes/sync-gateway-3-1-0-release-note.adoc:include::ROOT:whatsnew.adoc[tag=scopes-and-collections]`

Reply:
oddly this isn't showing in my editor 😕

Reply:
The diff (`-` sign and red background) suggests that it's been deleted in this commit.